src/App.css

```diff
@@ -1,3 +1,151 @@
-body {
-  background-color: midnightblue !important;
+h2 {
+  color: white !important;
+  margin-bottom: 0 !important;
+}
+
+.detail h4 {
+  color: #fff;
+}
+
+.details {
+  margin-top: 27px;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 10px;
+  margin-top: 30px;
+}
+
+div.grid a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.active-data {
+  background-color: #fd5294;
+  padding: 10px;
+}
+
+.inactive-data {
+  background-color: #ce4277;
+  padding: 10px;
+}
+
+h3 {
+  color: #fff !important;
+}
+
+div.container h2 {
+  color: rgb(39, 39, 39) !important;
+  padding-top: 10px;
+}
+
+.detail {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px;
+}
+
+.country {
+  background-color: #fd5294;
+}
+
+.total {
+  background-color: #ce4277;
+}
+
+.recover {
+  background-color: #ce4277;
+}
+
+.deaths {
+  background-color: #fd5294;
+}
+
+a img {
+  width: 27px;
+}
+
+section.top-page {
+  display: flex;
```

Comment:
- [ ] The previous code reviewer has asked you to ensure that your app's home page matches the design that was given. However, upon comparing the two designs, it appears that the background colors of the countries' list should differ from each other, just like in the given design.

### Your design
![image](https://github.com/ctokoli/react-capstone/assets/73854999/2e915d2e-184e-4570-83cb-f54c7910bde1)

### Given design
![image](https://github.com/ctokoli/react-capstone/assets/73854999/a4ef6aa4-ed8d-45a6-8c11-926251dc3748)